src/routers/api/room.router.js

```diff
@@ -0,0 +1,18 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const roomController = require('../../controllers/room.controllers');
+const { notemtyDate, notemtyId } = require('../../middleware/rooms.midleware');
+
+const router = express.Router();
+
+router.get('/', roomController.getRooms);
+
+router.get('/:id', roomController.searchRoom);
+
+router.post('/', roomController.createRoom);
+
+router.put('/:id', roomController.updateRoom);
```

Comment:
Chưa có validate cho input đầu vào của 2 thằng này. Sử dụng `joi` để làm. Viết 1 con middleware rồi truyền `schema` để joi nó tự validate.
Doc: [joi.dev](https://joi.dev/api/)
Example:
https://dev.to/itnext/joi-awesome-code-validation-for-node-js-and-express-35pk